src/app/core/service/cra-service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { CraModel } from '../model/cra-model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CraService {
+  constructor(private http: HttpClient) {}
+
+  createCra(cra: CraModel): Observable<CraModel> {
+    const rqt = this.http.post<CraModel>('/api/v1/activity', cra);
```

Comment:
its better to create a const for the URL string

Reply:
oui surtout quand on va ajouter de nouveau services